src/atoms/Video/Video.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Video = ({ video }) => {
+  function getVideoPlayer(video) {
+    let playerEmbed;
+    const uri = new URL(video.url);
+    switch (uri.hostname) {
+      case 'www.facebook.com':
+        playerEmbed = (
+          <iframe
+            title={video.credit.name}
+            src={video.url}
+            width="560"
+            height="460"
+            scrolling="no"
+            frameBorder="0"
+            allowtransparency="true"
+            allowFullScreen={true}
+          ></iframe>
+        );
+        break;
+      case 'player.vimeo.com':
+        playerEmbed = (
+          <video
+            id={video.credit.name}
+            autoPlay={true}
+            muted={true}
+            loop={true}
+          >
+            <source src={video.url} type="video/mp4" />
+            Your browser does not support HTML5 video.
+          </video>
+        );
+        break;
+      case 'mpr.apmcdn.org':
```

Comment:
It looks like vimeo and apmcdn have the exact same code.  Can we find a way to combine them.  Can you do ```case 'player.vimeo.com' || 'mpr.apmcdn.org'```  or can make a function to output the same thing?

Reply:
I tried this and it didn't work, however i will give it another go.